_maps/_basemap.dm

```diff
@@ -22,6 +22,10 @@
 		// SKYRAT EDIT END
 		#include "map_files\PubbyStation\PubbyStation.dmm"
 
+		//Tannhauser-Gate Maps
+		#include "map_files\AlphaStation\alphastation_lower.dmm"
```

Comment:
as far as i can tell you only need to specify 1 the primary z-lvel file

Reply:
In testing I had issues with it not loading the lower when the map was split into sperate levels like for Blueshift